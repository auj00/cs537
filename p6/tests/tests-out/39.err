rm: cannot remove 'mnt/file1': Function not implemented
