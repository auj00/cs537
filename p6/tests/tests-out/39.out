Correct
allocated datablocks on test-disk1: found 3 expected 1
